src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1196,3 +1196,14 @@ def test_great_expectations(pyi_builder):
         from great_expectations.render.view import view
         v = view.DefaultJinjaView()
         """)
+
```

Comment:
There must be two empty lines between individual tests (this is part of why linter check is failing).

Reply:
Added 2 empty lines